### PR TITLE
New memory maps implemented for h7 and f4

### PR DESCRIPTION
**Pull request by ShubhadaHarkut-eaton**
Memory map implemented for h7 and f4

### CODE REVIEW

**Review by prayassamriya — APPROVED** [review]

File: boot/zephyr/boards/nucleo_f429zi.overlay [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * Set the partitions with first MB to make use of the whole Bank1
+		 * module-test : internal flash for module test. can be used for logging as well when not in use (4 sectors 16kb each)
+		 */
+		boot_partition: partition@0 {
```
Comment (prayassamriya) [inline boot/zephyr/boards/nucleo_f429zi.overlay:20]: we will have to add read me file in boards directory with what and why information.

File: boot/zephyr/boards/nucleo_f429zi.overlay [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+			label = "mcuboot";
+			reg = <0x00000000 DT_SIZE_K(128)>;
+		};
+		slot0_partition: partition@20000 {
+			label = "image-0";
+			reg = <0x00020000 DT_SIZE_K(896)>;
+		};
+		scratch_partition: partition@100000 {
+			label = "image-scratch";
+			reg = <0x00100000 DT_SIZE_K(128)>;
```
Comment (VineetaNarkhede-eaton) [inline boot/zephyr/boards/nucleo_f429zi.overlay:30]: is this the maximum sector size in F429?
Please add comment in both files to convey that scratch should always be equal to biggest sector size in application area (slot0/slot1).

File: boot/zephyr/boards/nucleo_f429zi.overlay [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+/delete-node/ &storage_partition;
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/*
+		 * Set the partitions with first MB to make use of the whole Bank1
+		 * module-test : internal flash for module test. can be used for logging as well when not in use (4 sectors 16kb each)
```
Comment (VineetaNarkhede-eaton) [inline boot/zephyr/boards/nucleo_f429zi.overlay:18]: is this comment applicable here

**Review by VineetaNarkhede-eaton — CHANGES_REQUESTED** [review]
minor comments

File: boot/zephyr/boards/nucleo_f429zi.overlay (hunk 2 above)
Comment (ShubhadaHarkut-eaton) [reply boot/zephyr/boards/nucleo_f429zi.overlay:30]: added a generic comment to address the scratch size

File: boot/zephyr/boards/nucleo_f429zi.overlay (hunk 3 above)
Comment (ShubhadaHarkut-eaton) [reply boot/zephyr/boards/nucleo_f429zi.overlay]: fixed

**Review by VineetaNarkhede-eaton — APPROVED** [review]
Make sure to update the confluence page with new mapping and a note for scratch size.

**Pull request merged**